docs_user/modules/proc_rolling-back-the-control-plane-adoption.adoc

```diff
@@ -138,26 +130,23 @@ for i in {1..3}; do
 done
 ----
 
-. If the Ceph NFS service is running on the deployment as a {rhos_component_storage_file_first_ref}
-backend, you must restore the pacemaker ordering and colocation constraints
-involving the "openstack-manila-share" service:
+. If the Ceph NFS service is running on the deployment as a {rhos_component_storage_file_first_ref} back end, you must restore the Pacemaker order and colocation constraints for the `openstack-manila-share` service:
 +
 ----
 
-sudo pcs constraint order start ceph-nfs then openstack-manila-share kind=Optional id=order-ceph-nfs-openstack-manila-share-Optional
-sudo pcs constraint colocation add openstack-manila-share with ceph-nfs score=INFINITY id=colocation-openstack-manila-share-ceph-nfs-INFINITY
+$ sudo pcs constraint order start ceph-nfs then openstack-manila-share kind=Optional id=order-ceph-nfs-openstack-manila-share-Optional
+$ sudo pcs constraint colocation add openstack-manila-share with ceph-nfs score=INFINITY id=colocation-openstack-manila-share-ceph-nfs-INFINITY
 
 ----
 
-. Verify that the source cloud is operational again, e.g. by
-running `openstack` CLI commands or using the {dashboard_first_ref}.
-//kgilliga: Should we be more specific? Which CLI commands?
+. Verify that the source cloud is operational again, for example, you
```

Comment:
Maybe you could show an example command, i.e?

. Verify that the source cloud is operational again:

$ example command